Sources/Core/Sources/Render/RenderCoordinator.swift

```diff
@@ -62,9 +62,7 @@ public class RenderCoordinator: NSObject, RenderCoordinatorProtocol, MTKViewDele
     
     updateCamera(client.game.player, view)
     
-    guard
-      let transparentAndOpaqueCommandBuffer = commandQueue.makeCommandBuffer(),
-      let translucentCommandBuffer = commandQueue.makeCommandBuffer()
+    guard let worldRenderCommandBuffer = commandQueue.makeCommandBuffer()
```

Comment:
rename worldRenderCommandBuffer to just commandBuffer, there's only one command buffer so no need to specify